oidc-controller/api/authSessions/sessiontimeout.json

```diff
@@ -0,0 +1,5 @@
+{
+	"expired": 10,
+	"failed": 10,
+	"abandoned": 10
+}
```

Comment:
Do these get overridden by environment variables or helm values?
Or is this hard-coding, just in a json file that gets imported? In that case I'd just make them constants in the code to remove the extra file and the whole file import logic complication.

Reply:
The idea was that users could modify the file to extend different expiration times and for example add expiration for verified auth sessions. It was meant to be used similar to https://github.com/bcgov/vc-authn-oidc/blob/autodelete-auth-session/oidc-controller/api/logconf.json

Reply:
I should have read all the log statement rather than the first line only 🙂